README.md

```diff
@@ -51,166 +54,68 @@ libraryDependencies += "com.google.cloud" % "google-cloud-translate" % "1.94.2"
 
-Getting Started
----------------
-#### Prerequisites
-For this tutorial, you need a [Google Developers Console](https://console.developers.google.com/)
-project with "Translation API" enabled via the console's API Manager. You also need to enable
-billing via the [Google Developers Console](https://console.developers.google.com/).
+## About Cloud Translation
 
-Finally, you must set up the local development environment by
-[installing the Google Cloud SDK](https://cloud.google.com/sdk/) and running the following command
-in command line: `gcloud auth application-default login`. Alternatively, you can authenticate
-Translation requests using an API key. See
-[Translation quickstart](https://cloud.google.com/translate/v2/quickstart) for more details.
 
-#### Installation and setup
-You'll need to obtain the `google-cloud-translate` library. See the [Quickstart](#quickstart)
-section to add `google-cloud-translate` as a dependency in your code.
+[Cloud Translation][product-docs] can dynamically translate text between thousands of language pairs. Translation lets websites and programs programmatically integrate with the translation service.
 
-#### Creating an authorized service object
-To make authenticated requests to Google Translation, you must create a service object with
-credentials or use an API key. The simplest way to authenticate is to use
-[Application Default Credentials](https://developers.google.com/identity/protocols/application-default-credentials).
-These credentials are automatically inferred from your environment, so you only need the following
-code to create your service object:
+See the [Cloud Translation client library docs][javadocs] to learn how to
+use this Cloud Translation Client Library.
 
-```java
-import com.google.cloud.translate.Translate;
-import com.google.cloud.translate.TranslateOptions;
 
-Translate translate = TranslateOptions.getDefaultInstance().getService();
-```
 
-Notice that this code can be also used with an API key. By default, an API key is looked for in the
-`GOOGLE_API_KEY` environment variable. Once the API key is set, you can make API calls by invoking
-methods on the Translation service created via `TranslateOptions.getDefaultInstance().getService()`.
 
-You can also explicitly set the API key as follows:
-```java
-Translate translate = TranslateOptions.newBuilder().setApiKey("myKey").build().getService();
-```
+## Troubleshooting
 
-#### Detecting language
-With Google Translation you can detect the language of some text. The service will provide you with
-the code of the detected language and a level of confidence.
+To get help, follow the instructions in the [shared Troubleshooting document][troubleshooting].
 
-Add the following import at the top of your file:
+## Transport
 
-```java
-import com.google.cloud.translate.Detection;
-```
+Cloud Translation uses both gRPC and HTTP/JSON for the transport layer.
 
-Then pick a text sample:
```

Comment:
example

Reply:
restored examples in the custom content via `.readme-partials.yaml`